packages/components/src/mobile/picker/index.android.js

```diff
@@ -65,13 +65,12 @@ export default class Picker extends Component {
 			>
 				<PanelBody title={ title } style={ styles.panelBody }>
 					{ options.map( ( option, index ) => (
-						<>
+						<View key={ index }>
```

Comment:
It might be safer to use `option.label` as `key` or anything that is unique across options.

Reply:
Thanks for hint! Admittedly, the label should be unique, but just to be sure I mixed it with the index.

Reply:
They discourage usage of indexes as explained in their docs:

https://reactjs.org/docs/lists-and-keys.html#keys

> We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny’s article for an [in-depth explanation on the negative impacts of using an index as a key](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). If you choose not to assign an explicit key to list items then React will default to using indexes as keys.

We had a few performance issues in the past when using array indexes that's why I mentioned it. As far as I can tell, `<Picker />`'s options include `label` and `value`, I would bet they both should be unique. Are they any issues when you don't provide `index`?

Reply:
> Are they any issues when you don't provide index?

There should be no issues, will change it to `label` and `value` mix.